scripts/provisioning/testcmd.py

```diff
@@ -22,16 +22,22 @@
 
 from setupcmd import SetupCmd
 from cortx.utils.process import SimpleProcess
+from os import path
+
+SANITY_SCRIPT_PATH = '/opt/seagate/cortx/s3/scripts/s3-sanity-test.sh'
+LDAP_SWITCH = '-p'
+ENDPOINT_SWITCH = '-e'
 
 class TestCmd(SetupCmd):
   """Test Setup Cmd."""
   name = "test"
 
-  def __init__(self, config: str):
+  def __init__(self, config: str, plan: str):
     """Constructor."""
     try:
       super(TestCmd, self).__init__(config)
       self.read_ldap_credentials()
+      self.plan = plan
```

Comment:
what is the "plan" variable here. can we rename it as "test_plan"

Reply:
Done